src/utils/components/DiskModal/utils/helpers.ts

```diff
@@ -130,8 +130,12 @@ export const getDataVolumeFromState = ({
   resultVolume?: V1Volume;
   createOwnerReference?: boolean;
 }): V1beta1DataVolume => {
+  debugger;
```

Comment:
nice

Reply:
i would love to get stopped every time :) lol

Reply:
you would get stopped only if you open the dev tools, so we can say it's a feature lol

Reply:
removed thanks :)